CHANGELOG.md

```diff
@@ -24,9 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
-* Fix an issue where `--keep-workdirs` option for pytest was not available when running pytest without
```

Comment:
Why are we taking this out?

Reply:
This was already in 0.13.1 and was just copied over.